### DISCUSSION TITLE
关于不同系统下文件IO

### DISCUSSION THREAD

**Discussion by VansamaZWT** [liusy58/NJU_AdvancedProgramming_SP22#70] [Q&A] [answered]
因为上学期程序设计基础课程中课时不足，没能仔细讲解文件IO，且个人学习能力较差，没能掌握文件IO。想请问关于文件IO再windows和Linux系统下有不同吗？如果有，如何设计来使得程序可移植呢？

**Comment by RicoloveFeng** [comment]
_Search The Friendly Web_

**Comment by binbin00** [comment, marked as answer]
老师会在下周五的课讲授文件IO的内容，请认真请讲。
